packages/fxa-settings/README.md

```diff
@@ -1,75 +1,182 @@
+
+##### Component Classes
+
+When there is a pattern of commonly repeated utility combinations, it may make sense to extract this pattern into its own component or component utility class using Tailwind's `@apply`. A good rule of thumb for this is to consider if a pattern update would be expected to reflect in more than one component.
+
+For example:
+
+```tsx
+// my-component.tsx
+<button className="block w-full py-2 text-center mt-6 bg-grey-10 border border-grey-200 transition duration-150 rounded hover:border-grey-200 hover:bg-grey-100 hover:text-grey-400 hover:transition hover:duration-150 active:border-grey-400 active:bg-grey-100 active:text-grey-400 active:transition active:duration-150">
+  Click here!
+</button>
+
+// my-other-component.tsx
+<a href="#" className="block w-full py-2 text-center mt-6 bg-grey-10 border border-grey-200 transition duration-150 rounded hover:border-grey-200 hover:bg-grey-100 hover:text-grey-400 hover:transition hover:duration-150 active:border-grey-400 active:bg-grey-100 active:text-grey-400 active:transition active:duration-150">
+  Click here!
+</a>
+```
+
+This makes more sense as a new utility class instead of a component because sometimes we style a button and other times a link. Let's extract this instead into `ctas.scss` with a fitting name:
 
-Here are a few additional things to keep in mind when developing with Tailwind:
+```scss
+// ctas.scss
+.cta-neutral {
+  @apply block w-full py-2 text-center mt-6 bg-grey-10 border border-grey-200 transition duration-150 rounded;
 
-- We overwrite some of the default values provided by Tailwind, such as color and breakpoints. Refer to [tailwind.config.js](./tailwind.config.js) to see how values vary.
-- You do not need to import any custom or built-in styles/dependencies to use Tailwind. Just write your class names.
-- When it comes to time to create a production build we use PostCSS and PurgeCSS to strip out unused styles. PurgeCSS will look through all of the TSX files and identify which class names to keep styles for. In order for this to work properly it's important to avoid dynamic class names:
+  &:hover {
+    @apply border-grey-200 bg-grey-100 text-grey-400 transition duration-150;
+  }
 
-  ```tsx
-  // Bad
-  const Button = ({ textColor, children }) => {
-    return <button className={`text-${textColor}`}>{children}</button>;
-  };
-  <Button textColor="green-500">Hello, world</Button>;
+  &:active {
+    @apply border-grey-400 bg-grey-100 text-grey-400 transition duration-150;
+  }
+}
+```
+
+All we need to do now is `@import` ctas.scss into our `tailwind.css` file and apply our new utility class:
```

Comment:
It might make sense to note that you _cannot_ load PostCSS syntax directly into a React component (such as via `import './index.scss'`) and that it must be `@import`'d directly into a file that is parsed by PostCSS (in this case `tailwind.[s]css` or a file that is already imported).